introduction.md

```diff
@@ -2,3 +2,9 @@ hi, I used vi add introduction.md for this course
 this file content for practice github.
```

Comment:
Since you're becoming a GitHub master, could you also add "Killing history using git rebase" as one of your deadly skills?